src/backEnd/controllers/signup.js

```diff
@@ -0,0 +1,36 @@
+const bcrypt = require('bcryptjs');
+const { user } = require('../../database/models');
+
+exports.get = (req, res) => {
+  res.render('signup');
+};
+
+exports.post = (req, res) => {
+  const {
+    name, email, password, phone_number, facebook, repassword //eslint-disable-line
```

Comment:
confirm_password is better variable name instead of repassword

Reply:
fixed